### PR TITLE
Enabling the do-not-track feature of the vimeo video embed for amp articles

### PR DESCRIPTION
**Pull request by frj**
## What does this change?
This change enables the do-not-track feature for Vimeo videos embedded in amp articles.

For more details see the dnt flag in the Vimeo docs:

https://developer.vimeo.com/api/oembed/videos

> Whether to prevent the player from tracking session data, including cookies. Keep in mind that setting this argument to true also blocks video stats.

This change is dependent on a PR to the amphtml project:

https://github.com/ampproject/amphtml/pull/31406

Which has been merged but has not as yet been released.

In theory the PR should be tagged with 'PR Use: In Stable' once it is release for more details see:

https://amp.dev/documentation/guides-and-tutorials/learn/spec/release-schedule/#determining-if-your-change-is-in-a-release

## Why?

This is part of an ongoing effort to improve the privacy of users using the guardian products, in this case by stopping vimeo from collecting information on them through embedded vimeo videos.

### CODE REVIEW

File: src/amp/components/elements/VideoVimeoBlockComponent.tsx
Comment (oliverlloyd) [inline src/amp/components/elements/VideoVimeoBlockComponent.tsx:18]: I presume sending this attribute even if it's not supported won't have any adverse side effects?

Comment (frj) [reply src/amp/components/elements/VideoVimeoBlockComponent.tsx:18]: It shouldn't, but Im intending to hold off merging this change until the PR for the changes to amp-html are live.

**Review by oliverlloyd — APPROVED** [review]

File: src/model/json-schema.json
Comment (gtrufitt) [inline src/model/json-schema.json:2493]: Should this change be a part of this PR? It looks like a breaking change if not matched with Frontend cc/ @oliverlloyd

**Review by gtrufitt — COMMENTED** [review]
Looks like the AMP change has gone in so LGTM bar the query around CAPIPillar

File: src/model/json-schema.json
Comment (oliverlloyd) [reply src/model/json-schema.json:2493]: Not sure how this came about to be honest but I suspect if you merge in `main` it should go away?

Comment (frj) [reply src/model/json-schema.json]: Yes, Ive merged main and its gone away.

**Comment by frj** [comment]
The changes to amp-html are live, testing the changes locally and the do-not-track flag is seems to be working:

Screenshot 2021-01-12 at 15 55 03

**Review by shtukas — APPROVED** [review]

**Pull request merged**
